client/src/constants/participantTableConstants.js

```diff
@@ -162,6 +162,7 @@ const columns = {
   rosStartDate: { id: 'rosStartDate', name: 'Return of Service Start Date', sortOrder: 22 },
   rosSiteName: { id: 'rosSiteName', name: 'Site Name', sortOrder: 23 },
   employerName: { id: 'employerName', name: 'Hired By', sortOrder: 24 },
+  engagedBy: { id: 'engagedBy', name: 'Last Engaged By', sortOrder: 15.5 },
```

Comment:
What's `sortOrder` used for? I'm curious why it's so specific to 15.5

Reply:
Sort order determines which columns to show from left to right.  We should stick to whole numbers when using it, and just shift the others down by one.

Reply:
Good point, fixed that!